python/spear/game_world_service.py

```diff
@@ -14,3 +14,78 @@ def open_level(self, level_name):
 
     def set_game_paused(self, paused):
         self._rpc_client.call("game_world_service.set_game_paused", paused)
+
+    def find_actors(self):
+        return self._rpc_client.call("game_world_service.find_actors")
+
+    def find_actors_as_map(self):
+        return self._rpc_client.call("game_world_service.find_actors_as_map")    
+
+    def get_components(self, actor):
+        return self._rpc_client.call("game_world_service.get_components", actor)
+
+    def get_components_as_map(self, actor):
+        return self._rpc_client.call("game_world_service.get_components_as_map", actor)
+
+    def get_children_components(self, actor, include_all_descendants):
+        return self._rpc_client.call("game_world_service.get_children_components", actor, include_all_descendants)
+
+    def get_children_components_as_map(self, actor, include_all_descendants):
+        return self._rpc_client.call("game_world_service.get_children_components_as_map", actor, include_all_descendants)
+
+    def get_object_properties_as_string_from_object(self, object):
+        return self._rpc_client.call("game_world_service.get_object_properties_as_string_from_object", object)
+
+    def get_object_properties_as_string_from_struct(self, value_ptr, struct):
+        return self._rpc_client.call("game_world_service.get_object_properties_as_string_from_struct", value_ptr, struct)
+
+    def set_object_properties_from_string_for_object(self, object, string):
+        self._rpc_client.call("game_world_service.set_object_properties_from_string_for_object", object, string)
+
+    def set_object_properties_from_string_for_struct(self, value_ptr, struct, string):
```

Comment:
The names of the arguments here should match the C++ service, which should match `Unreal.h`.